scripts/install_tethys.sh

```diff
@@ -363,25 +363,13 @@ then
         git checkout ${BRANCH}
     fi
 
-    if [ -n "${DJANGO_VERSION}" ]
-    then
-        echo "Updating environment.yml Django version ${DJANGO_VERSION}..."
-        sudo sed -i.bak "s/django>=.*/django==${DJANGO_VERSION}/" "${TETHYS_SRC}/environment.yml"
-        sudo sed -i.bak "s/django>=.*/django==${DJANGO_VERSION}/" "${TETHYS_SRC}/micro_environment.yml"
-    fi
-
-    if [ -n "${PYTHON_VERSION}" ]
-    then
-        echo "Updating environment.yml Python version ${PYTHON_VERSION}..."
-        sudo sed -i.bak "s/django>=.*/python==${PYTHON_VERSION}/" "${TETHYS_SRC}/environment.yml"
-        sudo sed -i.bak "s/django>=.*/python==${PYTHON_VERSION}/" "${TETHYS_SRC}/micro_environment.yml"
-    fi
-
     if [ -n "${CREATE_ENV}" ]
     then
         # create conda env and install Tethys
         echo "Setting up the ${CONDA_ENV_NAME} environment..."
-        conda env create -n ${CONDA_ENV_NAME} -f "${TETHYS_SRC}/environment.yml"
+        sudo sed "s/python>=.*/python=${PYTHON_VERSION}/" "${TETHYS_SRC}/environment.yml" > "${TETHYS_SRC}/temp_environment.yml"
```

Comment:
Maybe change this to `generated_environment.yml` or something like that? Unless we are just getting rid of it? Temp just makes me think we need to get rid of it when we are done. But might be nice to keep it around so if we have any question as to what was used to generate the environment.

Reply:
This seems like a reasonable change. While the `temp_environment.yml` file is only needed during the environment setup, and, as far as I am aware, the only use case for the `install_tethys.sh` file is in the CI, it is conceivable that it would be used in another situation in which case `generated_environment.yml` file would more self-documenting.